scripts/sv/default_init.sh

```diff
@@ -15,5 +15,5 @@ if [[ ! "${ROLE}" == 'Master' ]]; then
     if  [[ ! $REFLOC =~ .+/$ ]]; then
       REFLOC+="/"
     fi
-    mkdir -p /reference && gsutil -m cp "$REFLOC"*.img /reference/
+    mkdir -p /mnt/1/reference && gsutil -m cp "$REFLOC"*.img /mnt/1/reference/
```

Comment:
Maybe add a comment explaining /mnt/1/ like you have in runWholePipeline here, just in case this fails if the default path changes in the future.